source/_integrations/amotion.markdown

```
@@ -0,0 +1,80 @@
+| `flow_eta_ref`            |       m<sup>3</sup>/h | Flow ETA request<br>Number between 600 and 3000   |
+| `dehum_h_req`            |       % |  Maximal acceptable humidity<br>Number between 0 and 100   |
+| `dehum_enabled`            |       True/False | Allow dehumidification<br>   |
+
+
+### Scenes
+
+Most devices also offer sets of preset behaviors, called scenes, which are uploaded as buttons when building an integration.
+
+### Functions
+
+Devices have something like programmable I/O. We call them “triggers.” Its behavior is similar to triggers in Home Assistant. A function is a switch to turn a trigger on or off (enable or disable a response to an input).
```

Markdownlint (0.37.0)

Files should end with a single newline character

(MD047, single-trailing-newline)